package.json

```diff
@@ -2867,4 +2891,4 @@
 		"jsonc-parser": "^2.3.0",
 		"lodash": "^4.17.19"
 	}
-}
+}
```

Comment:
There shouldn't be a change here. I think you took out the last new line character.

Reply:
fixed